src/lib/divider/divider.scss

```diff
@@ -0,0 +1,66 @@
+.mat-divider {
+  display: block;
+  margin: 0;
+  border-top-width: 1px;
+  border-top-style: solid;
+
+  &.mat-divider-vertical {
+    border-top: 0;
+    border-right-width: 1px;
+    border-right-style: solid;
+    position: static;
+    width: auto;
+  }
+
+  &.mat-divider-inset {
+    margin-left: 80px;
```

Comment:
For the inset's `margin` size, can you move the value (`80px`) into a variable at the top of the file?

Reply:
Done